pkg/master/master.go

```diff
@@ -47,9 +46,7 @@ import (
 	extensionsapiv1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
 	networkingapiv1 "k8s.io/kubernetes/pkg/apis/networking/v1"
 	policyapiv1beta1 "k8s.io/kubernetes/pkg/apis/policy/v1beta1"
-	rbacv1alpha1 "k8s.io/kubernetes/pkg/apis/rbac/v1alpha1"
```

Comment:
I think it's because the registry path doesn't include the version, so as long as the codec is able to decode (i.e., v1lapha1 is registered to api.Scheme), it's readable.